apps/docs/src/pages/_app.tsx

```diff
@@ -29,7 +29,10 @@ const KbarComponent = dynamic(() => import('../components/kbar'), {
 const Application: NextPage<AppProps<{}>> = ({ Component, pageProps }) => {
   const router = useRouter();
   const kbarActions = generateKbarActions(router);
-
+  
+  if(localStorage.getItem("theme") == undefined){
+    localStorage.setItem("theme","dark")
```

Comment:
Just to prevent the application to throw some error if localStorage is not defined on SSR.

```suggestion
  if(localStorage?.getItem("theme") == undefined){
    localStorage.setItem("theme","dark")
```

Reply:
> Just to prevent the application to throw some error if localStorage is not defined on SSR.